### ISSUE TITLE
any interest in compiling for node.js?

### ISSUE THREAD

**Issue by kevinmartinmauro** [louisstow/fann.js#1]
The fann library for node.js is pretty lame, and i think it's not so hard to compile for node.js if you've already compiled for emscripten. i just don't have the familiarity with C code to do it myself.

does this stackoverflow help? I'm willing to contribute as best I can
http://stackoverflow.com/questions/9020050/using-an-emscripten-compiled-c-library-from-node-js

**Comment by louisstow** [comment]
Yeah it's definitely possible. I would have thought Node.js bindings to the library would be better but I've not used it. Might just be a matter of assigning `module.exports = FANN;` at the end of the code.

**Comment by liesislukas** [comment]
It has dependency on "window" object. That's not present in Node.js environment. Trying to make it work with https://www.npmjs.com/package/mock-browser

added 2 things to fann.js:

first lines:
```
var MockBrowser = require('mock-browser').mocks.MockBrowser;


if (typeof window !== 'object') {
    window = MockBrowser.createWindow();
}
```

last line:
```
module.exports = FANN;
```

but still got errors, i may try it latter.
